js/main.js

```diff
@@ -43,28 +43,24 @@ $difficultyWrapper.addEventListener('click', handleDifficultyClicks);
 // HANDLE FORM
 function handleGameForm(event) {
   event.preventDefault();
-  // GET SESSION TOKEN
+  // GET SESSION TOKEN AND GET GAME URL
   var xhrToken = new XMLHttpRequest();
+  var xhrGame = new XMLHttpRequest();
```

Comment:
This variable should be in the same function block as the request it correlates to